servers/src/proto/journal_entry_protos.proto

```diff
@@ -0,0 +1,204 @@
+package tachyon.proto;
+option java_package = "tachyon.proto";
+
+// Wraps around all types of Tachyon journal messages.
+// next available id: 25
+message JournalEntry {
+  optional int64 sequence_number = 1;
+  oneof entry {
+    AddMountPointEntry add_mount_point = 2;
+    AsyncCompleteFileEntry async_complete_file = 3;
+    BlockContainerIdGeneratorEntry block_container_id_generator = 4;
+    BlockInfoEntry block_info = 5;
+    CompleteFileEntry complete_file = 6;
+    DeleteFileEntry delete_file = 7;
+    DeleteLineageEntry delete_lineage = 8;
+    DeleteMountPointEntry delete_mount_point = 9;
+    InodeDirectoryEntry inode_directory = 10;
+    InodeDirectoryIdGeneratorEntry inode_directory_id_generator = 11;
+    InodeFileEntry inode_file_entry = 12;
```

Comment:
all the other names don't say "_entry", except this one. Should "_entry" be removed to stay consistent?